docs/en/install_deploy.md

```diff
@@ -0,0 +1,393 @@
+# Install and Deploy
+
+## Software and Hardware Environment Requirements
+
+* Operating system: CentOS 7, Ubuntu 20.04, macOS >= 10.15. Where Linux glibc version >= 2.17. Other operating system versions have not been fully tested and cannot be guaranteed to function correctly.
```

Comment:
function correctly
=>
be fully compatible

Reply:
Ok, thank you.